recipes/novelrt/all/conanfile.py

```diff
@@ -0,0 +1,154 @@
+        "libsndfile:shared": True,
+        "openal:shared": True,
+        "PNG:shared": True,
+        "BZip2:shared": True,
+        "flac:shared": True,
+        "fmt:shared": True,
+        "onetbb:tbbproxy": False,
+        "onetbb:tbbmalloc": False,
+        "Opus:shared": True,
+        "Ogg:shared": True,
+        "Vorbis:shared": True,
+        "spdlog:header_only": True,
+        "vulkan-loader:shared": True,
+    }
+    cmake = None
+    unsupportedOS = ['AIX', 'Android', 'Arduino', 'Emscripten', 'FreeBSD', 'Macos', 'Neutrino', 'SunOS', 'WindowsCE', 'WindowsStore', 'iOS', 'tvOS', 'watchOS']
+    unsupportedCompilers = ['gcc', 'intel', 'mcst-lcc', 'qcc', 'sun-cc', 'msvc']
+
+    @property
+    def _compilers_minimum_version(self):
+        return {
+            "clang": "10",
+            "Visual Studio": "16"
+        }
+
+    @property
+    def _source_subfolder(self):
+        return "source"
+
+    @property
+    def _build_subfolder(self):
+        return "artifacts"
+
+    @property
+    def _is_static_lib_win(self):
+        return str(self.settings.compiler.runtime) in ["MTd", "MT"]
+
+    @property
+    def _is_debug_runtime_win(self):
+        return str(self.settings.compiler.runtime) in ["MDd", "MTd"]
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version], strip_root=True)
+
+    def configure_cmake(self):
+        cmakePath = self.deps_env_info["cmake"].CMAKE_ROOT
+        if cmakePath:
+            if self.settings.os == "Windows":
+                cmakePath += "\\bin\\cmake.exe"
+            else:
+                cmakePath += "/bin/cmake"
+            self.output.info(f"Overriding CMake to use version from Conan at: {cmakePath}")
+        cmake = CMake(self, cmake_program=cmakePath, parallel=True)
+        cmake.definitions["NOVELRT_BUILD_DOCUMENTATION"] = "Off"
+        cmake.definitions["NOVELRT_BUILD_TESTS"] = "Off"
+        cmake.definitions["NOVELRT_BUILD_SAMPLES"] = "Off"
+        cmake.definitions["NOVELRT_BUILD_INTEROP"] = "Off"
+        cmake.configure()
+        return cmake
+
+    def imports(self):
+        if self.settings.os == "Windows":
+            self.copy("*.dll", dst="thirdparty", src="bin")
+            self.copy("*.dll", dst="thirdparty", src="lib")
+
+    def build(self):
+        self.cmake = self.configure_cmake()
+        self.cmake.build()
+
+    def package(self):
+        self.copy("*LICENCE.md", dst="licenses", keep_path=False)
+        self.copy("include/NovelRT/*.h", dst="", keep_path=True)
+        self.copy("*include/NovelRT.h", dst="include", keep_path=False)
+        self.copy("*.spv", dst="bin/Resources/Shaders", keep_path=False)
+        if self.settings.os == "Windows":
+            self.copy("*NovelRT.lib", dst="lib", keep_path=False)
+            self.copy("*NovelRT.dll", dst="bin", keep_path=False)
+            self.copy("*bz2.dll", dst="bin", keep_path=False)
+            self.copy("*FLAC.dll", dst="bin", keep_path=False)
```

Comment:
it sounds weird... it shouldn't install DLLs of dependencies, like flac, ogg, vorbis, etc.